### ISSUE TITLE
When i added Mixin as a coremod i cant use my dev environment anymore

### ISSUE THREAD

**Issue by bebeli555** [SpongePowered/Mixin#425]
So after i added Mixin dependency when u try to launch the debug mode from eclipse. My mod doesnt load. The issue is that it cant find the mod in the run > mods. I have to manually build it and place it to there every time if i want to use it and i cant debug anymore. But otherwise it works fine its just the dev environment not working for some reason?

**Comment by Mumfrey** [comment]
This is an issue tracker for bugs in mixin not a support forum, please use discord for support queries.
